ibm_catalog.json

```diff
@@ -0,0 +1,442 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-activity-tracker",
+      "label": "Cloud automation for Activity Tracker",
```

Comment:
This is not the name that was in the GIT issue? Did OM want this name, or where did it come from? The issue has "Cloud automation for Activity Tracker Event Routing"